QUEST_JOBSTEP.tsv

```diff
@@ -2135,53 +2135,53 @@ QUEST_JOBSTEP_20150317_002134	Go meet Doppelsoeldner Master in Forgotten Port.
 QUEST_JOBSTEP_20150317_002135	Doppelsoeldner Master wants you to prove your skills by defeating Gaigalas in Flowerless Forest.
 QUEST_JOBSTEP_20150317_002136	Report to Doppelsoeldner Master
 QUEST_JOBSTEP_20150317_002137	Defeated Gaigalas. Report to Doppelsoeldner Master.
-QUEST_JOBSTEP_20150317_002138	Talk to Cronomancer Master
-QUEST_JOBSTEP_20150317_002139	Talk to Chronomancer Master in High Crooked Road.
+QUEST_JOBSTEP_20150317_002138	Talk to the Cronomancer Master
+QUEST_JOBSTEP_20150317_002139	Talk to the Chronomancer Master in High Crooked Road.
 QUEST_JOBSTEP_20150317_002140	Defeat Reaverpedes in Remaining Tree Dale
-QUEST_JOBSTEP_20150317_002141	For the test of Chronomancer Master, defeat Reaverpedes in Remaining Tree Dale.
-QUEST_JOBSTEP_20150317_002142	Report to Chronomancer Master
-QUEST_JOBSTEP_20150317_002143	Defeated Reverpede. Report to Cronomancer Master.
-QUEST_JOBSTEP_20150317_002144	Meet Wizard Master
+QUEST_JOBSTEP_20150317_002141	For the test of the Chronomancer Master, defeat Reaverpedes in Remaining Tree Dale.
+QUEST_JOBSTEP_20150317_002142	Report back to the Chronomancer Master
+QUEST_JOBSTEP_20150317_002143	You defeated enough Reaverpedes. Report back to the Cronomancer Master.
+QUEST_JOBSTEP_20150317_002144	Meet the Wizard Master
 QUEST_JOBSTEP_20150317_002145	Meet the Wizard Master in Klaipeda
-QUEST_JOBSTEP_20150317_002146	Get rid of the Ironbaum in Siauliai Eastern Woods
-QUEST_JOBSTEP_20150317_002147	Wizard Master is unsatisfied with the situtaion. No one is fighting against the monsters up front. Go to Siauliai Eastern Woods and defeat Ironbaum.
-QUEST_JOBSTEP_20150317_002148	Defeated Ironbaum. Go back to Wizard Master.
-QUEST_JOBSTEP_20150317_002149	Meet Pyromancer Master
-QUEST_JOBSTEP_20150317_002150	Meet Pyromancer Master in Klaipeda
-QUEST_JOBSTEP_20150317_002151	Pyromancer Master is not pleased with Cryomancer Master who isn't doing anything when Klaipeda is under threat. Defeat Ironbaum in Siauliai Eastern Woods and show him how it should be done.
-QUEST_JOBSTEP_20150317_002152	Defeated Ironbaum. Go back to Pyromancer Master.
+QUEST_JOBSTEP_20150317_002146	Get rid of the Ironbaums in Siauliai Eastern Woods
```

Comment:
Get rid of the Ironbaums -> Defeat Ironbaum
Siauliai Eastern Woods -> East Siauliai Woods